docs/user_guides/integrations/flink.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Documentation on how to configure an external Flink cluster to write features in to the Hopsworks Feature Store
+---
+# Flink Integration
+
+Connecting to the Feature Store from an external Flink cluster, such as AWS EMR and GCP DataProc requires configuring it with the Hopsworks certificates. This guide explains step by step how to connect to the Feature Store from an external Flink cluster.
+
+## Download the Hopsworks Certificates
+In the *Project Settings*, select the *integration* tab and scroll to the *Configure Spark Integration* section. Click on *Download certificates*. 
+
+<p align="center">
+    <figure>
+        <img src="../../../assets/images/guides/integrations/spark_integration.png" alt="Spark integration tab">
+        <figcaption>The Spark Integration gives access to Jars and configuration for an external Spark cluster</figcaption>
+    </figure>
+</p>
+
+Hopsworks uses X.509 certificates for authentication and authorization. If you are interested in the Hopsworks security model, you can read more about it in this [blog post](https://www.logicalclocks.com/blog/how-we-secure-your-data-with-hopsworks).
```

Comment:
Use this link instead: https://www.hopsworks.ai/post/how-we-secure-your-data-with-hopsworks